### DISCUSSION TITLE
Grott compatibility with new installation

### DISCUSSION THREAD

**Discussion by stefa168** [johanmeijer/grott#254] [Q&A] [answered]
Hi, I am currently renovating my home and in the coming months I will be allowed to turn on the system, which is composed of

- Two `SPH 10000TL3 BH-UP` with Growatt LiFePo batteries connected to each of them
- A Shinemaster connected via ethernet to my LAN

I was curious to know if Grott is currently compatible with this setup, and if not, what I could do to help to get it to work, as I'd prefer to keep my data locally (I am also running home assistant so another reason is to feed more data to it...)

Thanks!

**Comment by jupppo** [comment]
I have an SPH3600 with Batteries. That works perfectly

**Reply by stefa168** [reply]
And you're using a Shinemaster, right?

**Reply by jupppo** [reply]
yes worked with shinemaster, altough I switched back to Shine-S

**Reply by stefa168** [reply]
> yes worked with shinemaster, altough I switched back to Shine-S

@jupppo Out of curiosity, why did you switch to Shine-S?

**Reply by jupppo** [reply]
less energy usage :-) 
Had two inverters before and now have one. But I never tried grott with two inverters, just with the new one

**Comment by johanmeijer** [comment]
I think Grott should also work with a shinemaster without doing anything additional.

**Reply by stefa168** [reply]
Thanks! So it should work even with two inverters, right?

**Comment by johanmeijer** [comment, marked as answer]
Yes. I run with 2 inverters. 2 of the same type will work direct out of the box. Support for two different type inverters is available in the latest beta version.
